server/player/player.go

```diff
@@ -117,7 +119,9 @@ func New(name string, skin skin.Skin, pos mgl64.Vec3) *Player {
 		locale:   language.BritishEnglish,
 		scale:    *atomic.NewFloat64(1),
 	}
+	p.mc.Store(&entity.MovementComputer{Gravity: 0.08, Drag: 0.02, DragBeforeGravity: true})
 	p.pos.Store(pos)
+	p.vel.Store(mgl64.Vec3{})
```

Comment:
This should be the player's last velocity that they signed out with, meaning you'll have to save it in the player database when a player leaves

Reply:
Fixed.